examples/java/src/test/java/org/apache/beam/examples/cookbook/BigQueryStorageAPIStreamingIT.java

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
High level, do you want to offer user an option of not to use deduplication? If user don't need deduplication, they could use default stream and subject to less quota and higher performance on writes. It is in COMMITTED mode only and more like using the old insertall without insert_id.

Reply:
We could add this pretty easily, but I'd prefer to wait until there's a request. The rate of duplicates is likely to be higher than expected (especially with a runner like Dataflow that dynamically rebalances and retries largish chunks of work) so the savings would have to be significant to justify it IMO.